tests/integration/conftest.py

```diff
@@ -69,3 +69,15 @@ def local_pypiserver() -> Iterator[None]:
             raise RuntimeError("pypiserver didn't start fast enough")
 
         yield
+
+
+def pytest_runtest_setup(item: Any) -> None:
+    marks = set(mark.name for mark in item.iter_markers())
+    to_test = utils.determine_integration_tests_to_run_for("HEAD")
```

Comment:
It reads much better this way (since the function is named `determine_integration_tests_to_run_for`).

Reply:
I'll ask differently, since you're not proposing ref to be something we can override through an env variable for tests, do we need to call the function `determine_integration_tests_to_run_for` instead of `determine_integration_tests_to_run` ?

Reply:
I dropped `_for` in the latest revision.